roles/mirror_ocp_release/tasks/registry.yml

```
@@ -30,7 +30,7 @@
     --from={{ ocp_release_data['container_image'] | quote }}
     --to-release-image={{ mor_registry_url }}/{{ mor_registry_path }}:{{ mor_version }}
     --to={{ mor_registry_url }}/{{ mor_registry_path }}
-    --print-mirror-instructions={{ mor_is_type | lower }}
+    {{ mor_version is version("4.14", "<") | ternary("", "--print-mirror-instructions=" + mor_is_type | lower ) }}
```

nit: I tested and `--dry-run` will prevent any modification in the target registry.
```suggestion
    {{ mor_version is version("4.14", "<") | ternary("--dry-run", "--print-mirror-instructions=" + mor_is_type | lower ) }}
```